src/Uno.Extensions.Reactive.Generator/Bindables/BindableViewModelGenerator.cs

```diff
@@ -19,10 +20,7 @@ public BindableViewModelGenerator(BindableGenerationContext ctx)
 
 	private bool IsSupported(INamedTypeSymbol type)
 		=> _ctx.IsGenerationEnabled(type)
-			?? type
-				.Constructors
-				.SelectMany(ctor => ctor.Parameters)
-				.Any(parameter => _ctx.IsInputOrCommand(parameter.Type));
+			?? type.Name.EndsWith("ViewModel", StringComparison.Ordinal) && type.IsPartial();
```

Comment:
maybe it is best to offer an [GeneratedBindableVM(true/false)] attribute to customize this?

Reply:
It's the case just above. The "end by ViewModel" is the general case, but you do have an attribute to customize top match your needs.